pkgs/development/python-modules/fedpkg/fedpkg-buildfix.diff

```diff
@@ -0,0 +1,17 @@
+diff -ruN orig/setup.py new/setup.py
```

Comment:
Could you remove this line and the dates from the next two? It will make for less changes when the patch is updated. Also the file does not fix build, so `fix-paths.patch` would be a better name.

Reply:
sure, updated